x/cron/keeper/keeper.go

```diff
@@ -67,13 +69,14 @@ func (k *Keeper) Logger(ctx sdk.Context) log.Logger {
 // ExecuteReadySchedules gets all schedules that are due for execution (with limit that is equal to Params.Limit)
 // and executes messages in each one
 func (k *Keeper) ExecuteReadySchedules(ctx sdk.Context, executionStage types.ExecutionStage) {
-	telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), LabelExecuteReadySchedules)
+	startTime := time.Now()
```

Comment:
I still don't understand why do you need a separate variable `startTime` here.

Just calling an original method `telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), LabelExecuteReadySchedules)` via `defer` resolved the initial problem

Reply:
Gotcha. Changed